src/OrchardCore.Modules/OrchardCore.Workflows/Activities/CorrelateTask.cs

```diff
@@ -10,11 +10,13 @@ namespace OrchardCore.Workflows.Activities
     public class CorrelateTask : TaskActivity
     {
         private readonly IWorkflowScriptEvaluator _scriptEvaluator;
+        private readonly IWorkflowExpressionEvaluator _expressionEvaluator;
         private readonly IStringLocalizer S;
 
-        public CorrelateTask(IWorkflowScriptEvaluator scriptEvaluator, IStringLocalizer<CorrelateTask> localizer)
+        public CorrelateTask(IWorkflowScriptEvaluator scriptEvaluator, IWorkflowExpressionEvaluator expressionEvaluator, IStringLocalizer<CorrelateTask> localizer)
```

Comment:
@lampersky this what I mean, just we need to add this on the module docs

Reply:
@hishamco what do you exactly want to add there?

![image](https://user-images.githubusercontent.com/6403130/179374395-eb8e858c-14b4-4489-b16c-d2027908a3b9.png)

Reply:
No no, seems I was wrong, may be we need to add some docs [here](https://github.com/OrchardCMS/OrchardCore/blob/main/src/docs/releases/1.5.0.md)